django/utils/crypto.py

```diff
@@ -64,7 +66,12 @@ def get_random_string(length, allowed_chars=RANDOM_STRING_CHARS):
 
 def constant_time_compare(val1, val2):
     """Return True if the two strings are equal, False otherwise."""
-    return secrets.compare_digest(force_bytes(val1), force_bytes(val2))
+    warnings.warn(
+        "constant_time_compare() is deprecated. Use hmac.compare_digest() instead.",
+        RemovedInDjango61Warning,
```

Comment:
```suggestion
        RemovedInDjango70Warning,
```
(requires an import)